pkg/sfu/forwarder.go

```diff
@@ -340,40 +343,46 @@ func (f *Forwarder) DetermineCodec(codec webrtc.RTPCodecCapability, extensions [
 		}
 
 	case mime.MimeTypeVP9:
-		// DD-TODO : we only enable dd layer selector for av1/vp9 now, in the future we can enable it for vp8 too
-		isDDAvailable := ddAvailable(extensions)
-		if isDDAvailable {
+		if f.isReceiverSimulcast {
 			if f.vls != nil {
-				f.vls = videolayerselector.NewDependencyDescriptorFromOther(f.vls)
+				f.vls = videolayerselector.NewSimulcastFromOther(f.vls)
 			} else {
-				f.vls = videolayerselector.NewDependencyDescriptor(f.logger)
+				f.vls = videolayerselector.NewSimulcast(f.logger)
 			}
+			// TODO: Add temporal layer selector for VP9
```

Comment:
I usually add specific TODO so that it is easier to seartch.

This can either be `VP9-TODO` or even more specific like `VP9-SIMULCAST-TODO`. That allows me to fairly easily say what is missing still for VP9 Simulcast thee months down the line.

Reply:
Changed to `VP9-SIMULCAST-TODO` and `AV1-SIMULCAST-TODO`!